Webapp/sources/templates/email/reaction_approval_request.html

```diff
@@ -24,7 +24,7 @@
     <a href="{{ url_for("reaction_approval.request_response", token=token, _external=True, _scheme="protocol") }}">
         click here.</a>
 </p>
-<p>Alternatively, you can paste the following link in your browser"s address bar:</p>
+<p>Alternatively, you can paste the following link in your browsers address bar:</p>
```

Comment:
```suggestion
<p>Alternatively, you can paste the following link in your browser&#39;s address bar:</p>
```
html apostrophe symbol